src/main/java/mate/academy/intro/controller/OrderController.java

```diff
@@ -0,0 +1,81 @@
+package mate.academy.intro.controller;
+
+import io.swagger.v3.oas.annotations.Operation;
+import io.swagger.v3.oas.annotations.tags.Tag;
+import jakarta.validation.Valid;
+import java.util.List;
+import lombok.RequiredArgsConstructor;
+import mate.academy.intro.dto.order.CreateOrderRequestDto;
+import mate.academy.intro.dto.order.OrderDto;
+import mate.academy.intro.dto.order.OrderItemDto;
+import mate.academy.intro.dto.order.UpdateOrderStatusRequestDto;
+import mate.academy.intro.model.User;
+import mate.academy.intro.service.order.OrderService;
+import org.springframework.data.domain.Pageable;
+import org.springframework.http.HttpStatus;
+import org.springframework.security.access.prepost.PreAuthorize;
+import org.springframework.security.core.Authentication;
+import org.springframework.validation.annotation.Validated;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PatchMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.ResponseStatus;
+import org.springframework.web.bind.annotation.RestController;
+
+@Tag(name = "Order management", description = "Endpoints for managing orders")
+@RequiredArgsConstructor
+@Validated
+@RestController
+@RequestMapping("/api/orders")

```

Comment:
```suggestion
@RequestMapping("/orders")
```

use `server.servlet.context-path=/api` in application.properties